products/AOS-CX-Switching

```diff
@@ -0,0 +1,210 @@
+    support: 2025-11-25
+    release: 2020-11-30
+    latest: "10.06.01"
+    lts: false
+    discontinued: true
+    cycleShortHand: ""
+    link: https://www.arubanetworks.com/support-services/end-of-life/
+    codename: nothing
+    auto: false
+  - releaseCycle: "10.05.xxxx"
+    releaseLabel: ""
+    eol: 2021-09-21
+    support: 2021-09-21
+    release: 2020-06-30
+    latest: "10.06.01"
+    lts: false
+    discontinued: true
+    cycleShortHand: ""
+    link: https://www.arubanetworks.com/support-services/end-of-life/
+    codename: nothing
+    auto: false
+  - releaseCycle: "10.04.xxxx"
+    releaseLabel: ""
+    eol: 2021-09-21
+    support: 2021-09-21
+    release: 2019-11-1
+    latest: "10.06.01"
+    lts: false
+    discontinued: true
+    cycleShortHand: ""
+    link: 
+    codename: nothing
+    auto: false
+  - releaseCycle: "10.03.xxxx"
+    releaseLabel: ""
+    eol: 2021-09-21
+    support: 2021-09-21
+    release: 2019-06-19
+    latest: "10.06.01"
+    lts: false
+    discontinued: true
+    cycleShortHand: ""
+    link: https://www.arubanetworks.com/support-services/end-of-life/
+    codename: nothing
+    auto: false
+
+# Set an icon for the product from https://simpleicons.org/
+# If the icon is not available on simpleicons, set it to "NA"
+# As an example, https://simpleicons.org/?q=opensuse links to
+# https://simpleicons.org/icons/opensuse.svg and https://simpleicons.org/icons/opensuse.pdf
+# So the slug is `opensuse` (the SVG filename without extension).
+iconSlug: ministryofmagic
+
+# A few extra fields define overall page behaviour
+
+# URL for the page
+permalink: /timeturner
+
+# A list of alternate URLs that will redirect to the permalink. This is nice to let people use easier to remember URLs. For eg, we redirect /golang to /go
+alternate_urls:
+  - /hourglass
+
+# More information link. This link should contain
+# information about the release policy and schedule
+# This is NOT the product URL
+# Do not use a localized URL (such as one containing en-us) if possible
+releasePolicyLink: https://jkrowling.com/timeturner-releases
+
+# Whether to hide the "Active Support" column (optional, default true)
+activeSupportColumn: false
+
+# Whether to hide/show the latest release column. If the product doesn't have patch releases, set this to false. (optional, default true)
+releaseColumn: true
+
+# Whether to show the release date column
+# optional, default false
+releaseDateColumn: true
+
+# What to call the End of Life  (Security Support) column. (optional)
+eolColumn: Service Status
+
+# Whether to hide/show the discontinued column. Set to true, if you're tracking a device. This usually means the device is no longer available for sale or is no longer being manufactured. Set discontinued: true/false inside a release.
+
+discontinuedColumn: false
+
+# Command that can be used to check the current version. (optional)
+command: swish and flick
+
+# An image that shows a graphical representation of the releases.
```

Comment:
And please remove all comments which really not needed so make this MR/PR more readable please